flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -287,7 +287,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         const onlineFeaturesStatus = NXDataStore.get("CONFIG_ONLINE_FEATURES_STATUS", "UNKNOWN");
 
         if (onlineFeaturesStatus === "UNKNOWN") {
-            CDU_OPTIONS_TELEX.ShowPage(this);
+            new NXNotif().showNotification({ message: "You have not yet configured the TELEX option. To do so, go to the electronic flight bag's settings page and set the TELEX setting on the ATSU/AOC tab. More information can be found at https://docs.flybywiresim.com/telex" });
```

Comment:
**thought:** Should we not show the dialog immediately like we did before ?

Reply:
I thought about it but thought some might find it intrusive.

Reply:
> 
> 
> I thought about it but thought some might find it intrusive.

That was intentional. 😛

Reply:
> > I thought about it but thought some might find it intrusive.
> 
> That was intentional. 😛

Ok, sorry about that; I will make the change.